src/mpi/pt2pt/mpir_request.c

```diff
@@ -569,8 +570,15 @@ int MPIR_Grequest_progress_poke(int count,
             {
 		mpi_errno = (request_ptrs[i]->u.ureq.greq_fns->poll_fn)(request_ptrs[i]->u.ureq.greq_fns->grequest_extra_state, &(array_of_statuses[i]));
                 if (mpi_errno) MPIR_ERR_POP(mpi_errno);
+		if MPIR_Request_is_complete(request_ptrs[i]) made_progress = 1;
```

Comment:
Having the `( )` around the function call would be nice too.

Reply:
agreed

Reply:
fixed